models/CurveType.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Coinbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: |
+  CurveType is the type of cryptographic curve associated with a PublicKey. 
+
+  * secp256k1 = SEC compressed 33-bytes.
+  * edwards25519 =  y (255-bits) || x-sign-bit (32-bytes).
```

Comment:
I think the way this is documented is a little bit confusing which is my bad for stating it like this in the other doc. 

This reads like the public key is 255-bits + 32-bytes which isn't true. It's 255-bits + 1-bit for a total of 32-bytes. It may be worth clarifying that here.

It also could be beneficial to link to documentation on where these encodings are described elsewhere.

For Ed25519, it's page 6 of the paper here: https://ed25519.cr.yp.to/ed25519-20110926.pdf
For secp256k1, the compressed encoding is here: https://secg.org/sec1-v2.pdf#subsubsection.2.3.3